Clicker/View Controllers/DraftsViewController.swift

```diff
@@ -14,6 +14,9 @@ protocol FillsDraftDelegate {
 
 class DraftsViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
     
+    // MARK: Constants
+    let draftCellID = "draftCellID"
```

Comment:
if you want (optional), you should move this to a Constants.ReuseIdentifiers struct.